Invalid typings of the first parameter of `firestore.onChange` - optional before, after

<!-- DO NOT DELETE 
validate_template=true
template_path=.github/ISSUE_TEMPLATE/---report-a-bug.md
-->

### Related issues

None found.

### [REQUIRED] Version info

**node:** `v10.15.1`

**firebase-functions:** `v2.2.0`

**firebase-tools:** `v6.3.1`

**firebase-admin:** `^7.0.0`

### [REQUIRED] Test case 

```typescript
import { firestore } from 'firebase-functions';

firestore
  .document('collection/{documentId}')
  .onUpdate(change => change.after.data());
```

### [REQUIRED] Steps to reproduce

Try to compile the above snippet using the latest version of TypeScript compiler (`v3.3.0` at the time) with `strictNullChecks` enabled.

Compilation will fail with the following error:
```
test.ts:5:23 - error TS2532: Object is possibly 'undefined'.

5   .onUpdate(change => change.after.data());
                        ~~~~~~~~~~~~


Found 1 error.
```

### [REQUIRED] Expected behavior

Typings should not define `before` nor `after` as optional, according to the [docs](https://firebase.google.com/docs/reference/functions/functions.firestore.DocumentBuilder#onUpdate):
```
function(non-null functions.Change containing non-null functions.firestore.DocumentSnapshot, optional non-null functions.EventContext)
Event handler which is run every time data is updated in Cloud Firestore.
Value must not be null.
```

### [REQUIRED] Actual behavior

See `Steps to reproduce`.

### Were you able to successfully deploy your functions?

Not applicable.

--- a/changelog.txt
+++ b/changelog.txt
@@ -1,0 +1,1 @@
+fixed - improved types of the `Change` class to describe both `before` and `after` fields as non-optional

--- a/src/cloud-functions.ts
+++ b/src/cloud-functions.ts
@@ -90,7 +90,7 @@ export interface EventContext {
  * to the event, "after" represents the state after the event.
  */
 export class Change<T> {
-  constructor(public before?: T, public after?: T) {}
+  constructor(public before: T, public after: T) {}
 }
 
 /** ChangeJson is the JSON format used to construct a Change object. */

